client/src/components/Gallery.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import $ from 'jquery';
+
+class Gallery extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      featuredRestaurant: ''
+    };
+  }
+
+  getRequest(){
+    var restID = Math.floor(Math.random()*100+1).toString().padStart(3, '0');
+    $.ajax({
+      url: '/restaurants/${restID}',
+      success: function(data){
+        console.log(data);
+      },
+      error: function(err){
+        console.log(err);
+      },
+      contentType: 'application/json',
+      dataType: 'json'
+    });
+  }
+
+  componentDidMount(){
+    this.getRequest()
+  }
+
+  render(){
+    return (
+      <div class="photo-container">
+        <div class="one"></div>
```

Comment:
I have two comments about the way you are using "class" here:
1. In JSX, `class` should be `className`.
2. Instead of "class" here, I think `id` is more appropriate. For classes, you have multiple components that share similar attributes. For `id`, it is considered an unique item. You can consider `className="picture" id="one"`.

Reply:
[https://css-tricks.com/the-difference-between-id-and-class/](url)

Reply:
Thanks I'll make the changes in my next version.